src/tcp.c

```diff
@@ -3382,7 +3383,11 @@ relpTcpSend(relpTcp_t *const pThis, relpOctet_t *const pBuf, ssize_t *const pLen
 		if(written == -1) {
 			switch(errno_save) {
 				case EAGAIN:
+#if defined(_AIX)
+				// AIX Workarround handling other ERRNO codes
+				case 0:
 				case EINTR:
```

Comment:
This encapsulates case EINTR: within defined(_AIX), is this intended?

Reply:
> This encapsulates case EINTR: within defined(_AIX), is this intended?

Yes as far as I know we do not have "EINTR" in errno on other systems, typically it is set to "EAGAIN".